apps/src/lab2/views/components/predict.module.scss

```diff
@@ -27,3 +27,9 @@
   background-color: $light_white;
   border-radius: 5px;
 }
+
+.resetError {
+  color: $red;
```

Comment:
I wonder if we should use a DSCO alert instead of red text.

Reply:
Good idea!